guides/k8-akka-cluster/src/main/paradox/index.md

```diff
@@ -0,0 +1,465 @@
+
+```bash
+mvn clean package docker:build
+```
+
+
+#### Single-module project
+
+Follow this section if your application is built using a single-module Maven project, i.e. single `pom.xml` located at the root of the project's directory.
+
+Add the following plugin settings on the `pom.xml` under project root directory to register the fabric8 Maven plugin:
+
+```
+<plugin>
+    <groupId>io.fabric8</groupId>
+    <artifactId>docker-maven-plugin</artifactId>
+    <version>0.20.1</version>
+    <configuration>
+        <skip>false</skip>
+        <images>
+            <image>
+                <name>%g/%a:%l</name>
+                <build>
+                    <from>openjdk:8-jre-alpine</from>
+                    <tags>
+                        <tag>latest</tag>
+                        <tag>${project.version}</tag>
+                    </tags>
+                    <assembly>
+                        <descriptorRef>artifact-with-dependencies</descriptorRef>
+                    </assembly>
+                    <entryPoint>
+                        java -cp '/maven/*' -DactorSystemName=${AKKA_ACTOR_SYSTEM_NAME} -Dakka.actor.provider=cluster -Dakka.remote.netty.tcp.hostname="$AKKA_REMOTING_BIND_HOST" -Dakka.remote.netty.tcp.port="$AKKA_REMOTING_BIND_PORT" -Dakka.cluster.seed-nodes.0="akka.tcp://${AKKA_ACTOR_SYSTEM_NAME}@${AKKA_SEED_NODE_HOST}:${AKKA_SEED_NODE_PORT}" com.mycompany.MainClass
+                    </entryPoint>                    
+                </build>
+            </image>
+        </images>
+    </configuration>
+</plugin>
+```
+
+When building a container image for the application, the base image [openjdk:8-jre-alpine](https://hub.docker.com/_/openjdk/) will be used to provide a working JRE 8 installation for the application to run on. The container image name expression is `%g/%a:%l` where `%g`, `%a`, and `%l` are Maven project's group id, artefact id, and the image tag. When published, our image will tagged with `latest` and `${project.version}`
+
+The fabric8 Maven plugin will place all the jar files under `/maven` directory which need to be added to the JVM's classpath. Also note that we have added the Akka clustering related system properties as part of the JVM options.
+
+Replace `com.mycompany.MainClass` with the actual main class of your application.
+
+Execute the following command to containerize and publish your application's Docker image:
+
+```bash
+mvn clean package docker:build
+```
+
+### Creating Kubernetes Service for Akka remoting
+
+Next we will declare the Akka remoting port we'd like to expose from our Pod using Kubernetes Service so it can be referenced from one Pod to another.
+
+Execute the following command to create Kubernetes Service for the application's Akka remoting port:
+
+```bash
+cat << EOF | kubectl create -f -
+{
+  "apiVersion": "v1",
+  "kind": "Service",
+  "metadata": {
+    "labels": {
+      "app": "myapp"
+    },
+    "name": "myapp-akka-remoting"
```

Comment:
The name will show up in `kubectl get services`, i.e.

```
Felixs-MBP-2:activator-lagom-java-chirper felixsatyaputra$ kubectl get services
NAME                            CLUSTER-IP   EXTERNAL-IP   PORT(S)                      AGE
activityservice                 None         <none>        9000/TCP                     1s
activityservice-akka-remoting   10.0.0.46    <none>        2551/TCP                     2s
cassandra                       None         <none>        9042/TCP                     13d
chirpservice                    None         <none>        9000/TCP                     1s
chirpservice-akka-remoting      10.0.0.204   <none>        2551/TCP                     1s
friendservice                   None         <none>        9000/TCP                     1s
friendservice-akka-remoting     10.0.0.229   <none>        2551/TCP                     1s
kubernetes                      10.0.0.1     <none>        443/TCP                      13d
nginx-default-backend           10.0.0.94    <none>        80/TCP                       13d
nginx-ingress                   10.0.0.213   <pending>     80:31470/TCP,443:32396/TCP   13d
web                             10.0.0.53    <none>        9000/TCP                     1s
```

I think it's good to be explicit about which service is exposing what port(s) so they can be updated/deleted independently of others.